package.json

```diff
@@ -21,13 +21,14 @@
     "build:ui": "cross-env NODE_ENV=production webpack --colors --config webpack.renderer.config.js --mode=production --profile --json > app/dist/build.stats.json",
     "start": "concurrently -k \"yarn frontend\" \"yarn backend\"",
     "start:new": "concurrently -k \"yarn frontend\" \"yarn backend:new\"",
+    "serve": "http-server -p 8081 ./app/dist",
```

Comment:
wouldn't be ok to use our yarn frontend while developing? and on circle run against the artifacts